build/publishNewVersion.mjs

```diff
@@ -26,8 +26,8 @@ const BUMP_TYPES = ['major', 'minor', 'patch', 'prerelease'];
 const program = new Command();
 program
     .option('--dry', 'dry run', false)
-    .option('--tag', 'tag to use on NPM', 'latest')
-    .option('--branch', 'allow deploy on branch', 'master')
+    .option('--tag <tag>', 'tag to use on NPM', 'latest')
```

Comment:
When not specifying any parameter, commander assumes the argument is of type boolean.